apps/wallet-dashboard/components/dialogs/unstake/views/UnstakeView.tsx

```diff
@@ -156,14 +159,30 @@ export function UnstakeView({
             </DialogLayoutBody>
 
             <DialogLayoutFooter>
+                {isNotEnoughGas && (
+                    <div className="pt-sm">
+                        <InfoBox
+                            supportingText={GAS_BUDGET_ERROR_MESSAGES[NOT_ENOUGH_BALANCE_ID]}
```

Comment:
I would better use this
```suggestion
                            supportingText={GAS_BUDGET_ERROR_MESSAGES[GAS_BALANCE_TOO_LOW_ID]}
```